src/App.tsx

```diff
@@ -1,173 +1,110 @@
-import { Loader } from './components/Loader';
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { useEffect, useState } from 'react';
+import {
+  Navigate, NavLink, Route, Routes,
+} from 'react-router-dom';
+import cn from 'classnames';
+import { getPeople } from './api';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { Person } from './types/Person';
+
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const getPeopleFromServer = async () => {
+    try {
+      const dataFromServer = await getPeople();
+
+      const peopleWithParents = dataFromServer.map((person: Person) => {
+        return {
+          ...person,
+          mother: dataFromServer.find(
+            (p: Person) => p.name === person.motherName,
+          ),
+          father: dataFromServer.find(
+            (p: Person) => p.name === person.fatherName,
+          ),
+        };
+      });
+
+      setPeople(peopleWithParents);
+    } catch {
+      (
+        <p data-cy="peopleLoadingError" className="has-text-danger">
+          Something went wrong
+        </p>
+      );
+    }
```

Comment:
It doesn't work such way :) 
Let's create a state with error and show this paragraph only when we have error